internal/tcmu/commands.go

```diff
@@ -0,0 +1,79 @@
+package torustcmu
+
+import (
+	"github.com/coreos/go-tcmu"
+	"github.com/coreos/go-tcmu/scsi"
+)
+
+func (h *torusHandler) handleSyncCommand(cmd *tcmu.SCSICmd) (tcmu.SCSIResponse, error) {
+	clog.Debugf("syncing")
+	err := h.file.Sync()
+	if err != nil {
+		clog.Errorf("sync failed: %v", err.Error())
+		return cmd.MediumError(), nil
+	}
+	return cmd.Ok(), nil
+}
+
+func (h *torusHandler) handleReportDeviceID(cmd *tcmu.SCSICmd) (tcmu.SCSIResponse, error) {
+	v := h.name
+	if len(h.name) > 240 {
```

Comment:
are these const coming from a spec? probably we should either link to the spec here or add some comments on the const?